src/lib/options/options.models.ts

```diff
@@ -5,10 +5,12 @@ export interface OptionsAction extends Action {
   payload: Partial<Options>;
 }
 
+export type DataTransformType = null | ((data: Data[] | WideData[]) => Data[] | WideData[]);
```

Comment:
we do not need to export this separately to refer to it.
you may just use `Options["dataTransform"]`